core/src/test/java/org/nem/core/model/NemesisBlockTest.java

```diff
@@ -50,7 +50,7 @@ public void nemesisBlockCanBeCreated() {
 			MatcherAssert.assertThat(block.getTimeStamp(), IsEqual.equalTo(TimeInstant.ZERO));
 
 			// 2 multisig aggregate transactions
```

Comment:
update comment

Reply:
one?

Reply:
I thought there were only transfer transactions? 🤔

Reply:
you seem to be checking that there is a fee for an multisig, so i assume there is one still?

```
MatcherAssert.assertThat(block.getTotalFee(), IsEqual.equalTo(EXPECTED_MULTISIG_AGGREGATE_FEE));
```

Reply:
The ``EXPECTED_MULTISIG_AGGREGATE_FEE`` is zero.  I left the check for a couple of reasons.
1. to verify that there is no multisig fee
2. if a multisig is added in a future testnet release, this will fail.